apps/backend/src/database/migrations/1681495109117-migration.ts

```diff
@@ -0,0 +1,71 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class migration1681495109117 implements MigrationInterface {
+  name = 'migration1681495109117';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
```

Comment:
Have all of these alter columns been tested with data? 

is the casting of data working by default for existing data? Has it been validated?

Reply:
Updated: using ::type 
Tested with existing db data